sim.c

```diff
@@ -249,7 +249,7 @@ static void oper_poke_and_stun(Glyph* restrict gbuffer, Mark* restrict mbuffer,
 
 #define MOVEMENT_CASES                                                         \
   'N' : case 'n' : case 'E' : case 'e' : case 'S' : case 's' : case 'W'        \
-      : case 'w'
+                                                               : case 'w'
```

Comment:
Not sure why clang-format made this change here. Are you sure it's reading the `.clang-format` file for this project?

Reply:
i'm new to clang-format, i just used `clang-format -i sim.c`, from the root of Orca-c, shouldn't it work like this?

Reply:
ok, i should have used `clang-format -style=file -i sim.c` instead, sorry

Reply:
Huh. Ok, I noticed two things:
1. The existing header include order was wrong. `sim.h` is supposed to be at the top. I don't know how it became moved down.
2. I updated my version of `clang-format`, and now it wants to move `sim.h` back to the top. But it doesn't do the thing with moving the `case 'w'` to the right like that (which is obviously wrong.)

I think both of these hunks should just be removed.